.travis.yml

```diff
@@ -30,6 +31,12 @@ before_script:
   - mongo --version
   - mongod --version
   - snyk test
+# adding influxdb repository
```

Comment:
@chmac i guess i didn't put much thought into this. Anyways that link is just a repository for installation. I'm not sure how the influxdb would be used in tests nor whether it is necessary. Didn't think about the possibility of faking it, as @nicksellen already pointed out somewhere (i believe). i'll just get rid of this for now.